Model/Cache/Type/AdyenCache.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2021 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+// phpcs:disable Generic.CodeAnalysis.UselessOverridingMethod.Found
+
+namespace Adyen\Payment\Model\Cache\Type;
+
+use Magento\Framework\App\Cache\Type\FrontendPool;
+use Magento\Framework\Cache\Frontend\Decorator\TagScope;
+
+class AdyenCache extends TagScope
+{
+    /**
+     * Cache type code unique among all cache types
+     */
+    const TYPE_IDENTIFIER = 'adyen';
+
+    /**
+     * The tag name that limits the cache cleaning scope within a particular tag
+     */
+    const CACHE_TAG = 'adyen-webhook';
```

Comment:
I would keep this description more generic, so that it can be used for other purposes

```suggestion
    const CACHE_TAG = 'adyen';
```